src/reanimated2/utils.ts

```diff
@@ -32,3 +32,14 @@ export function isSharedValue<T>(value: any): value is SharedValue<T> {
   'worklet';
   return value?._isReanimatedSharedValue === true;
 }
+
+/**
+ * @param val preferred value
+ * @param min minimum allowed value
+ * @param max maximum allowed value
+ * Clamps given value within a range of values between a defined minimum bound and a maximum bound.
+ */
+export function clamp(val: number, min: number, max: number) {
```

Comment:
No need for extra abbreviation - although I do love the idea of 3-letters variables.

```suggestion
export function clamp(value: number, min: number, max: number) {
```